### DISCUSSION TITLE
Feature request - Duplicate Document incl. nested documents

### DISCUSSION THREAD

**Discussion by cisoLKT** [outline/outline#3683] [Ideas]
Hi there!

Outline already includes the functionality to duplicate a single document. 

One helpful feature would be the ability to duplicate a document including all nested documents (just like the normal duplicate functionality, also in the menu of the parent document). This would be useful for e.g. copying onboarding templates for new colleagues.

**Comment by tommoor** [comment]
Yep, we have an outstanding request for this – makes a lot of sense 👍
